sorc/chgres_cube.fd/run/config.C768.odin.hrrr.nml

```diff
@@ -0,0 +1,42 @@
+# Takes lots of memory.  Use two nodes - six tasks per node.
```

Comment:
@JeffBeck-NOAA I removed this run directory a while back as I did not want scripts and config files in the sorc directory.

Reply:
@GeorgeGayno-NOAA, I pushed a commit to remove this directory.